### PR TITLE
Update organizations doc for Console with info about membership requirements

### PR DESCRIPTION
**Pull request by praneetloke**

### CODE REVIEW

File: themes/default/content/docs/intro/console/organizations.md
Comment (cnunciato) [inline themes/default/content/docs/intro/console/organizations.md:135]: We have a `notes` shortcode that would be good here, I think:

```suggestion
{{% notes %}}
Regardless of the identity your org uses, org membership is always invite-only. Only the creator of the org gets automatic access. Everyone else must be invited by an admin to be admitted into the org.
{{% notes %}}

```

File: themes/default/content/docs/intro/console/organizations.md
Comment (cnunciato) [inline themes/default/content/docs/intro/console/organizations.md:131]:
```suggestion
When you add a new org to Pulumi, it uses the Pulumi identity, by default. This means the membership is
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (cnunciato) [inline themes/default/content/docs/intro/console/organizations.md:137]:
```suggestion
However, if want your org to mirror an org on a third party identity service such as GitHub, GitLab, or Bitbucket, you can
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (cnunciato) [inline themes/default/content/docs/intro/console/organizations.md:148]:
```suggestion
and go to the **Settings** page. Click on the **Access Management** navigation tab to the left and scroll down to the
```

**Review by cnunciato — APPROVED** [review]
This all looks great to me, thanks for writing it! Just a couple minor suggestions -- tagging @davidwrede also in case he has any others.

File: themes/default/content/docs/intro/console/organizations.md
Comment (praneetloke) [inline themes/default/content/docs/intro/console/organizations.md:137]:
```suggestion
However, if you want your org to mirror an org on a third party identity service such as GitHub, GitLab, or Bitbucket, you can
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (praneetloke) [inline themes/default/content/docs/intro/console/organizations.md:138]:
```suggestion
```

**Comment by praneetloke** [comment]
Thanks, @cnunciato! @davidwrede I'll hold off merging this until you've had a chance to glance at this well.

File: themes/default/content/docs/intro/console/organizations.md
Comment (praneetloke) [inline themes/default/content/docs/intro/console/organizations.md:136]:
```suggestion
{{% /notes %}}
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (lukehoban) [inline themes/default/content/docs/intro/console/organizations.md:131]:
```suggestion
When you add a new org to Pulumi, it uses the Pulumi identity by default. This means the membership is
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (lukehoban) [inline themes/default/content/docs/intro/console/organizations.md:140]: Perhaps "as discussed in []()" with a link to the anchor?  (Just in case this content gets reorganized in the future).

Comment (praneetloke) [inline themes/default/content/docs/intro/console/organizations.md:140]:
```suggestion
change the backing identity. Enterprises can also choose SAML as the backing identity provider for an org as discussed in [SAML Single Sign-on](#saml-single-sign-on-sso). Changing your org's backing identity essentially changes the membership requirements you place
```

File: themes/default/content/docs/intro/console/organizations.md
Comment (susanev) [inline themes/default/content/docs/intro/console/organizations.md:149]: thanks for doing this work! a few important notes..

- i did a pass to remove "Click" from the console docs a bit ago as its pointer specific. we are using "select" now instead.
- i also believe we are trying to use "navigate" instead of "go to".
- it is also important that we avoid directional words, as folks that use screen readers do not have the same information about where things are placed on the page directionally.

```suggestion
To switch an organization's backing identity you must be an organization admin. Navigate to the organization's **Settings** page. Then navigate to **Access Management** and select the **Membership Requirements** button to get started with changing your organization's identity.

let me know if you have thoughts on a good place for this kind of documentation for future work.

```

**Review by davidwrede — COMMENTED** [review]
Small nit

File: themes/default/content/docs/intro/console/organizations.md
Comment (davidwrede) [reply themes/default/content/docs/intro/console/organizations.md:149]: @susanev Thanks for submitting this suggestion! I was in the middle of writing up the same changes, but saw that you beat me to it. :)

Comment (praneetloke) [reply themes/default/content/docs/intro/console/organizations.md:149]: > let me know if you have thoughts on a good place for this kind of documentation for future work.

I can add it to the README of this repo with this PR.

Comment (praneetloke) [reply themes/default/content/docs/intro/console/organizations.md]: Done. Updated the PR

**Comment by praneetloke** [comment]
Thank you all for the review! I'll merge this once the latest PR build passes.

**Pull request merged**
